### DISCUSSION TITLE
Removing Excessive fonts from the  Font Family dropdown.

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1328] [Q&A] [answered]
How can we remove all these fonts that are a pain when doing lots of work with the site. And to be just plain honest I DO NOT WANT my editors being able to choose but a certain limited number of fonts.

![image](https://user-images.githubusercontent.com/6786299/119721740-1cc0bb80-be31-11eb-95b0-250c8d0029d1.png)

**Comment by Gaganbird** [comment, marked as answer]
Simply remove the following code from theme.editor.json (\Portals\_default\vThemes\Basic\)

"Fonts": [
    {
      "Guid": "de28b274-ccef-4b8d-b8be-bb82aec413d9",
      "Name": "Arial Black",
      "Family": "Arial Black, Gadget, sans-serif",
      "Css": ""
    },
    ...

**Reply by scottw12** [reply]
Thanks:-)

**Reply by meetmandeep** [reply]
The answer provided by @Gaganbird only applies if you upgraded to RC 2. Going forward, you can easily add/remove fonts from Site > Design > Theme > Settings > Custom > Settings > Fonts
